### ISSUE TITLE
Client error: `GET https://api.github.com/%5Crepos%5C:vendor%5C:repoName%5Ccommits?sha=1.0` resulted in a `404 Not Found`

### ISSUE THREAD

**Issue by netbounty** [codedge/laravel-selfupdater#113]
Hey,
I want to execute the code to check if there are any updates and it gave me this kind of error message back. I need help..

**Comment by netbounty** [comment]
![Unbenannt](https://user-images.githubusercontent.com/65421448/82827258-eb747f00-9eae-11ea-9292-779c6ed80789.PNG)

**Comment by netbounty** [comment]
In my .env file:
![Unbenannt2](https://user-images.githubusercontent.com/65421448/82827359-2aa2d000-9eaf-11ea-8dcc-e140581bda65.PNG)

**Comment by codedge** [comment]
What is the URL to your Github repository? I cannot find any netbounty/shop repo on Github.

**Comment by netbounty** [comment]
Have a look in my profile, I have made it public now.

**Comment by netbounty** [comment]
But I already tried it out making it public and fetching, same errors appears

**Comment by codedge** [comment]
You specified a branch `v1.0` but you don't have a branch name `v1.0`. Remove the line for specifying the branch and it should work.

**Comment by netbounty** [comment]
So you mean changing `v1.0` to `1.0`? Or leaving it just empty?

**Comment by codedge** [comment]
Remove this line `SELF_UPDATER_USE_BRANCH="1.0"` from your `.env` file.

TLDR;
A [branch](https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell) is not a [release](https://git-scm.com/book/en/v2/Git-Basics-Tagging).

**Comment by netbounty** [comment]
![Unbenannt](https://user-images.githubusercontent.com/65421448/82838093-68aded00-9ecb-11ea-9625-2a14bcc3d8ef.PNG)
Now I'm getting this error. Sorry for asking but I really don't know what to do

**Comment by netbounty** [comment]
![Unbenannt](https://user-images.githubusercontent.com/65421448/82838466-9a738380-9ecc-11ea-83ae-b9a82d9f2a07.PNG)

**Comment by netbounty** [comment]
config/self-update.php
![Unbenannt](https://user-images.githubusercontent.com/65421448/82845218-f72e6880-9ee3-11ea-9ec4-765127cf958d.PNG)
.env
![Unbenannt2](https://user-images.githubusercontent.com/65421448/82845221-f7c6ff00-9ee3-11ea-95e8-6c20c321a6bd.PNG)
routes/web.php
![Unbenannt3](https://user-images.githubusercontent.com/65421448/82845222-f85f9580-9ee3-11ea-838c-2121e304a5ff.PNG)
Error page. **but why...** I really want to use this package and I need it. Help me please
![Unbenannt4](https://user-images.githubusercontent.com/65421448/82845289-35c42300-9ee4-11ea-9a11-bb4707e9cdd2.PNG)

**Comment by johnnybaida** [comment]
how did you solve? I get the same error with a private repository.

SELF_UPDATER_VERSION_INSTALLED="0.1"
SELF_UPDATER_SOURCE="github"
SELF_UPDATER_REPO_VENDOR="vendor-name"
SELF_UPDATER_REPO_NAME="repo-name"
SELF_UPDATER_USE_BRANCH=master
SELF_UPDATER_GITHUB_PRIVATE_ACCESS_TOKEN=token-code

**Comment by netbounty** [comment]
I gave up using this package. I've written my own self-updater working with a update server in behind.

**Comment by salahhusa9** [comment]
@johnnybaida ,i have the same probleme `Client error: `GET https://api.github.com/%5Crepos%5Csalahhusa9%5Cselfupdate%5Ccommits?sha=main` resulted in a `404 Not Found` response: {"message":"Not Found","documentation_url":"https://docs.github.com/rest"}`
